### ISSUE TITLE
[v23] Potential XSS fix

### ISSUE THREAD

**Issue by jeffersoncasimir** [aces/Loris#9480]
The following is a patch that can potentially address the XSS vulnerability of the request_account page:

Patch for `modules/login/templates/form_requestaccount.tpl`:
```
--- form_requestaccount.tpl (original)
+++ form_requestaccount.tpl (patched)
@@ -5,9 +5,9 @@
 <div class="panel panel-default panel-center">
   <div class="panel-heading">
     <h3 class="panel-title">
-        {$page_title}
+        {$page_title|escape}
     </h3>
   </div>
   <div class="panel-body">
       {if $success}
@@ -17,9 +17,9 @@
       <a href="/" class="btn btn-primary btn-block">
         Return to Login Page
       </a>
-      <b class="text-danger">{$form.firstname.error}</b>
+      <b class="text-danger">{$form.firstname.error|escape}</b>
       <b class="text-danger">{$form.lastname.error|escape}</b>
       <b class="text-danger">{$form.from.error|escape}</b>
+      <b class="text-danger">{$form.site.error|escape}</b>
```

**Comment by christinerogers** [comment]
related to [this report](https://mailman.bic.mni.mcgill.ca/pipermail/loris-dev/2024-September/001201.html)

**Comment by kongtiaowang** [comment]
This is another way to fix this issue. fix the XSS attack issue in 23.0-release #9481

**Comment by jeffersoncasimir** [comment]
Replaced by above
